.github/drafts/stale.yml

```
@@ -10,13 +10,16 @@ jobs:
       - uses: actions/stale@main
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-          stale-pr-message: 'Hello there,
-          we noticed that this PR has been inactive for a while now. If there are any changes which are suitable for our repository,
-          we would love to have it. It would be great if you can continue with this PR, but if you cannot, we might fork your changes and merge the changes ourselves.
+          stale-pr-message: "Hello there,
+          we have noticed that this PR has been inactive for a while now. If there are any changes which are suitable for our repository,
+          we would love to have it. It would be great if you can continue with this PR, but if you can't, we might fork your changes and merge the changes ourselves.
 
-          Since GitHub tracks contributions by commits, you will still be credited.
+          Since GitHub tracks contributions by commits, you'll still be credited.
+          Let us know what do you think about this. :slight_smile:
 
-          Let us know what you think 😃'
+
+          Feel free to join our Discord server for support, hangouts, and more: https://discord.gg/hScy8KWACQ
```

Inline link maybe? :)

```suggestion
          Feel free to [join our Discord server](https://discord.gg/hScy8KWACQ) for support, hangouts, and more
```